site/docs/guide/reactions.md

```diff
@@ -0,0 +1,168 @@
+# Reactions
+
+Bots can work with message reactions.
+There are two types of reactions, emoji reactions and custom emoji reactions.
+
+## Reacting to Messages
+
+Bots can add a single emoji reaction to a message.
+If a [Telegram Premium](https://telegram.org/faq_premium?setln=en) user added a custom emoji reaction to a message, bots can add the same reaction to this message, too.
```

Comment:
> A custom emoji reaction can be used if it is either already present on the message or **_explicitly allowed by chat administrators_**.\[1\]

[1]: https://core.telegram.org/bots/api#setmessagereaction ^reaction parameter description

Reply:
```suggestion
Bots can add a single emoji reaction to a message.
They can add custom emoji reactions only if it is explicitly allowed by the chat's administrators, or there is a custom emoji reaction previously added by a subscriber of [Telegram Premium](https://telegram.org/faq_premium?setln=en).
```

Reply:
Right, I'll clarify it.